modules/persistent-storage-hostpath-about.adoc

```diff
@@ -10,3 +10,19 @@
 In a production cluster, you would not use hostPath. Instead, a cluster administrator would provision a network resource, such as a GCE Persistent Disk volume, an NFS share, or an Amazon EBS volume. Network resources support the use of storage classes to set up dynamic provisioning.
 
 A hostPath volume must be provisioned statically.
+
+[IMPORTANT]
+====
+Do not mount the hostPath to the container root, `/`, as it can corrupt your host system if the container is sufficiently privileged. Do not mount any path on the host to a path the same name as the container, because the path in the container might have changes that should not be added to the host. It is safe to mount the host by using `/host`. For example:
+
+[source,yaml]
+----
+ ...
+      volumes:
+      - name: host-slash
+        hostPath:
+          path: /host
+          type: ''
+ ...
```

Comment:
I don't believe this is going to work as expected.  You need to define the volume from the host to be mounted and then the mount point in the container.

I think the snippet below would be better, as it shows the `/` directory from the host being mounted into the container at `/host` in a complete Pod definition:

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: test-host-mount
spec:
  containers:
  - image: registry.access.redhat.com/ubi/ubi8
    name: test-container
    volumeMounts:
    - mountPath: /host
      name: host-slash
  volumes:
   - name: host-slash
     hostPath:
       path: /
       type: ''
```